content/livesync/connector/index.textile

```diff
@@ -0,0 +1,336 @@
+product: livesync
+---
+
+The Database Connector uses the "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html to broadcast application-specific change events from your database to frontend clients via "Ably Channels":/channels.
+
+When you update the data in your database, you record a change event transactionally in a dedicated "outbox table":/livesync/connector/database#outbox. The Database Connector detects new changes in the outbox and publishes them as messages on Ably Channels. Using the Models SDK, your client applications subscribe to these change events and update their local state with the new data.
+
+The Database Connector is available as a "Docker":/https://www.docker.com/ container image that can be deployed on any cloud infrastructure platform that supports running Docker container images.
+
+The following is an example of how to pull the Database Connector image:
+
+```[sh]
+docker pull ghcr.io/ably-labs/adbc:latest
+```
+
+<aside data-type='note'>
+<p>Please "reach out":https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewform if you would rather use an Ably-hosted version.</p>
+</aside>
+
+h2(#docker). Deploy using Docker Compose
+
+Use "Docker Compose":https://docs.docker.com/compose/ to run the Database Connector locally for development purposes. This will create and run a PostgreSQL database and an instance of the Database Connector on your local machine.
+
+h3(#docker-setup). Setup Docker Compose
+
+To begin, create a @docker-compose.yml@ file with the following contents:
+
+```[yaml]
+version: '3'
+
+services:
+ adbc:
+   image: ghcr.io/ably-labs/adbc:latest
+   env_file:
+     - adbc.env # load config from env file
+
+   ## Uncomment below if you want to load config from your adbc.yaml file,
+   ## which takes precendence over config from the env.
+   # volumes:
+   #   - ./adbc.yaml:/adbc.yaml:ro # mount yaml config file
+
+   depends_on:
+     postgres:
+       condition: service_healthy
+   networks:
+     adbc_network:
+      postgres:
+   image: postgres:11-alpine
+   ports:
+     - 5432:5432
+   environment:
+     POSTGRES_USER: postgres
+     POSTGRES_PASSWORD: postgres
+     POSTGRES_DB: postgres
+   healthcheck:
+     test: ["CMD", "pg_isready", "-q", "-d", "postgres", "-U", "postgres"]
+     interval: 2s
+     retries: 30
+   networks:
+     adbc_network:
+   volumes:
+     - adbc_postgres_data:/var/lib/postgresql/data
+
+volumes:
+ adbc_postgres_data:
+
+networks:
+ adbc_network:
+```
+
+h3(#docker-configure). Configure in Docker Compose
+
+There are three "options":#options to configure the Database Connector. The following examples will use environment variables.
+
+Create a file called @adbc.env@ with at least the following configuration options:
+
+```[sh]
+ADBC_ABLY_API_KEY={{API_KEY}}
```

Comment:
This is interpolating an actual API key when I preview the docs site... is this expected?

![image](https://github.com/ably/docs/assets/9295665/e34b10f9-f3d6-4eb0-b087-46716e6f3ec2)